pyroma/tests.py

```diff
@@ -122,8 +123,9 @@ def test_only_config(self):
         self.assertEqual(
             rating,
             (
-                8,
+                7,
                 [
+                    "You should specify what Python versions you support with 'python_requires=....'.",
```

Comment:
```suggestion
                    (
                        "You should specify what Python versions you support with "
                        "the 'requires-python'/'python_requires' metadata."
                    )
```

Update to match above